src/components/DropZone/DropZone.tsx

```diff
@@ -363,18 +371,29 @@ export class DropZone extends React.Component<CombinedProps, State> {
       return;
     }
 
-    let size = 'extraLarge';
-    const width = this.node.getBoundingClientRect().width;
-
-    if (width < 100) {
-      size = 'small';
-    } else if (width < 160) {
-      size = 'medium';
-    } else if (width < 300) {
-      size = 'large';
+    let height;
+    let width;
+    const wrapperHeight = this.node.getBoundingClientRect().height;
+    if (wrapperHeight < Size.Small) {
+      height = Size.Small;
+    } else if (wrapperHeight < Size.Medium) {
+      height = Size.Medium;
+    } else if (wrapperHeight < Size.Large) {
+      height = Size.Large;
+    } else {
+      height = Size.ExtraLarge;
     }
-
-    this.setState({size});
+    const wrapperWidth = this.node.getBoundingClientRect().width;
```

Comment:
If we only need the width, should we use `this.node.offsetWidth`?

Reply:
The tests, she didn't like this suggestion.